app/build.gradle.kts

```diff
@@ -75,7 +75,15 @@ android {
       initWith(getByName("debug"))
       setMatchingFallbacks("debug")
     }
+    create("fdroid") { // Configuration for F-Droid flavor
+      // dimension = "default"
+      initWith(getByName("release"))
+      matchingFallbacks += "release"
+      applicationIdSuffix = ".fdroid"
+      versionNameSuffix = "-fdroid"
+    }
   }
+
```

Comment:
>fdroidImplementation(Libs.objectbox_gradle_plugin) {
    exclude(group = "io.objectbox", module = "objectbox-gradle-plugin")
  }
This code removes the deps from the fdroid build. As we're still depending on the objectbox on the other codes. We'll remove it after the end of the migration.

Reply:
OK, so how the app works without them?

Reply:
We're only excluding in the fdroid build. Other than that pretty much default for all default configurations. If we wanna build a new flavour it'll come out from default.

Reply:
@kelson42 At this stage we can't build fully for the fdroid as we haven't migrated all the funcality. As soon we've migrated all the stuff & little more build configuration we can build successfully for the fdroid.

Reply:
@gouri-panda The overall approach is to consider`develop` branch potentially ready to release any time. Therefore the rule is to not merged features which are halfway done as far as possible. Here is there is no reason to introduce now a feature which is halfway done: this is why I said "do that once at the end of the Room project".

Reply:
Okay, so we should remove this code from this PR because we will not publish our application on F-Droid until we successfully remove the Object Box from our codebase. F-Droid does not allow us to publish our app with the Object Box code. Additionally, we don't need this code for the Play Store build and other platform build types. Therefore, it is unused in this context. Please let me know if my explanation makes sense to you, @kelson42 and @gouri-panda.

Reply:
I have removed the fdroid variant and its dependencies from this PR.